apps/retrospective/constants.go

```diff
@@ -25,3 +25,20 @@ const (
 	GoodRating
 	NotableRating
 )
+
+// Resolution ...
+type Resolution int8
```

Comment:
Make constants for the list of resolution types.

Reply:
done